app/typer/sanity/sanity.ts

```diff
@@ -0,0 +1,51 @@
+import { PortableTextBlock } from '@portabletext/types';
+
+export interface SanityDokument {
+  _createdAt: string;
+  _rev: string;
+  _type: string;
+  _id: string;
+  api_navn: ESanityApiKey;
+  visningsnavn: string;
+  /*  nb: CustomSanityTyper.CUSTUM_BLOCK;
+  nn: CustomSanityTyper.CUSTUM_BLOCK;
+  en: CustomSanityTyper.CUSTUM_BLOCK; */
+}
+
+/* export enum CustomSanityTyper {
+  CUSTUM_BLOCK = 'customBlock',
+} */
+
+export enum ESanityApiKey {
+  TITTEL = 'tittelPaForside',
+  PUNKTLISTE = 'punktlisteMedEndringsgrunner',
+  TITTEL_PUNKTLISTE = 'tittelPunktlisteMedEndringsgrunner',
+}
+
+export type TextBlock = Record<ESanityApiKey, string> & {
+  api_navn: string;
+  [key: string]: unknown;
+};
+
+export interface IForsideTekstinnhold {
+  tittel: LocaleRecordString;
+  punktliste: LocaleRecordBlock;
+}
+
+//import { LocaleType } from '@navikt/familie-sprakvelger';
+export enum LocaleType {
+  en = 'en',
+  nb = 'nb',
+  nn = 'nn',
+}
+
+//Ikke brukt enda, men notert at det er gjort i eksempel repo
+export type LocaleRecordString = Record<LocaleType, string> & {
+  api_navn: string;
+  [key: string]: unknown;
+};
```

Comment:
Tror ikke denne vil være nødvendig for dere fordi det kun er brukt BlockContent i Sanity og ikke "vanlige input-felter" alle steder hvor det er koblet til en LocaleType

Reply:
Den er fjernet nå [2343bbb](https://github.com/bekk/nav-familie-endringsmelding/pull/13/commits/2343bbb3576b2b40055fcf696c663f31d3e6c282)